ci/playbooks/templates/deployment.ha.yaml.j2

```diff
@@ -38,6 +37,7 @@ kubernetes:
       user_name: admin
       user_password: admin
       user_id: admin
+  node_user: {{ sudo_user | default('root') }}
```

Comment:
may want to switch to 'ubuntu' or 'ec2-user' as the default user

Reply:
The playbook converting this template will be replacing with 'ubuntu'